railgun/internal/ctrlutils/utils.go

```diff
@@ -59,3 +60,24 @@ func IsAPIAvailable(mgr ctrl.Manager, obj client.Object) (bool, error) {
 
 	return true, nil
 }
+
```

Comment:
Whatever implementation choices get made in this PR, please add a comment to all functions dealing with ingress classes in the `annotations` package that they're deprecated in favor of &lt;specific replacement function name&gt;.

Or reuse them.

Reply:
:+1: 3dbb54261963e216f1fa391daa15555d35301b2b